.github/workflows/spell-checker.yml

```
@@ -0,0 +1,42 @@
+name: Spell Checker
+on:
+  pull_request_target:
+    branches:
+    - "**"
```

We only need 2 branch targets here.
```suggestion
    - "main"
    - "page"
```